gr-uhd/grc/gen_uhd_usrp_blocks.py

```diff
@@ -377,6 +377,21 @@
 		<type>\$type.type</type>
 		<nports>\$nchan</nports>
 	</$sourk>
+#if $sourk == 'source'
+	<source>
+		<name>rx_metad</name>
```

Comment:
That's why GRC has a button to disable port names, and I doubt the 3 chars make such a h differe.